{
    "id": "LUCENE-232",
    "title": "Version 1.3 reports IOException when re-creating an index",
    "details": {
        "labels": "",
        "priority": "Major",
        "components": [
            "core/index"
        ],
        "type": "Bug",
        "fix_versions": [],
        "affect_versions": "None",
        "resolution": "Fixed",
        "status": "Closed"
    },
    "description": "Version: Lucene 1.3 final \nError reported when I am (re-)doing an initialization on the index created \npreviously:\njava.io.IOException: couldn't delete _26a.f1\n\nThe problem disappearred after a re-start of the jvm, some files may be locked \nafter the index writer action !\nProblem does not appear in Version 1.2.",
    "attachments": {},
    "issue_links": {},
    "comments": [
        {
            "date": "2004-06-23T12:45:39+0000",
            "content": "Please give the latest 1.4 Release Candidate a try.  You problem may already be\nfixed in the the latest code. ",
            "author": "Otis Gospodnetic",
            "id": "comment-12321732"
        },
        {
            "date": "2004-07-10T04:09:51+0000",
            "content": "No reply, so let's assume it's fixed in 1.4.  ",
            "author": "Daniel Naber",
            "id": "comment-12321733"
        }
    ]
}